include/eve/arch/abi_of.hpp

```diff
@@ -65,6 +65,11 @@ namespace eve
             else                                   return emulated_{};
           }
         }
+        else if constexpr( spy::simd_instruction_set == spy::rvv_
+                           && spy::simd_instruction_set.width >= 64 )
+        {
+          return riscv_rvv_dyn_ {};
```

Comment:
What I mean is: instead of different kind of tags for different vlen (as it is done in ARM SVE), I use the only one tag: `riscv_rvv_dyn_`. Dynamic in this place means that the same code should work if you compile with vlen equals 64, 128, 256... (Though for now I'm testing only variant with vlen=128).

How about `riscv_rvv_vlen_`?